charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -4,11 +4,14 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
     PAYMENTS_QUEUE_MAX_DELIVERY_COUNT: "5"
+    S2S_NAME: "bulk_scan_payment_processor"
+    S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   image: 'https://hmctspublic.azurecr.io/bulk-scan/payment-processor:latest'
   keyVaults:
     "bulk-scan":
       resourceGroup: bulk-scan
       secrets:
+        - s2s-secret-payment-processor
```

Comment:
How about just `s2s-secret`?

Reply:
That long name was needed on S2S terraform side, as it deals with all service's secrets. To us it's just a secret for s2s.

Reply:
it's duplicated by s2s-secret which is already in bulk scan vault

Reply:
aaa, ok